### PR TITLE
Add running notary server instruction.

### PR DESCRIPTION
**Pull request by yuroitaki**

### CODE REVIEW

**Review by heeckhau — COMMENTED** [review]
Looks good.

I feel like it is worth mentioning that port 7047 should be open on the firewall

**Review by heeckhau — APPROVED** [review]

**Review by heeckhau — COMMENTED** [review]
Good improvements thanks!

After reading your latest changes, I think it would be better to talk about the keys and certificates first. They will need these files before they can run the server.
So it would be better to list these files first with a short explanation of what each required file exactly is.

File: src/developers/notary_server.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Run a Notary Server
+
+This guide shows you how to run a [notary server](https://github.com/tlsnotary/tlsn/tree/dev/notary-server) in an Ubuntu server instance.
+
+## Configure Server Setting
+All the following setting can be configured in the [config file](https://github.com/tlsnotary/tlsn/blob/dev/notary-server/config/config.yaml).
```
Comment (heeckhau) [inline src/developers/notary_server.md:6]:
```suggestion
All the following settings can be configured in the [config file](https://github.com/tlsnotary/tlsn/blob/dev/notary-server/config/config.yaml).
```

File: src/developers/notary_server.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+1. One *MUST* change the path of the following files, as by default dummy fixtures are used which is not secured at all
```
Comment (heeckhau) [inline src/developers/notary_server.md:8]:
```suggestion
1. Before running a notary server you need the following files. The default dummy fixtures are for testing only and should never be used in production.
```

File: src/developers/notary_server.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+    | File | Purpose | File Type | Compulsory to change |
+     ----- | ------- | ------------------ | -------------------- |
+    | TLS private key | Private key used for notary server's TLS certificate to establish TLS connection with prover | Compatible TLS private key in PEM format | Yes unless TLS is turned off |
+    | TLS certificate | Notary server's TLS certificate to establish TLS connection with prover | Compatible TLS certificate in PEM format | Yes unless TLS is turned off |
+    | Notary signature private key | Private key used for notary server's signature on the generated transcript of the TLS session with prover | A P256 elliptic curve private key in PEM format | Yes |
+    | Notary signature public key | Public key used for notary server's signature on the generated transcript of the TLS session with prover | A public key (in PEM format) that corresponds to the private key above | Yes |
```
Comment (heeckhau) [inline src/developers/notary_server.md:15]:
```suggestion
    | TLS private key | The private key used for the notary server's TLS certificate to establish TLS connections with provers | Compatible TLS private key in PEM format | Yes unless TLS is turned off |
    | TLS certificate | The notary server's TLS certificate to establish TLS connections with provers | Compatible TLS certificate in PEM format | Yes unless TLS is turned off |
    | Notary signature private key | The private key used for the notary server's signature on the generated transcript of the TLS sessions with provers | A P256 elliptic curve private key in PEM format | Yes |
    | Notary signature public key | The public key used for the notary server's signature on the generated transcript of the TLS sessions with provers | A matching public key (in PEM format) | Yes |
```

Comment (heeckhau) [reply src/developers/notary_server.md:15]: What does "compatible" mean here?

**Review by heeckhau — APPROVED** [review]

File: src/developers/notary_server.md (hunk 3 above)
Comment (yuroitaki) [reply src/developers/notary_server.md]: > What does "compatible" mean here?

Ah just wanted to mean it has to be key/cert that the TLS spec allows, will drop the term

**Review by heeckhau — APPROVED** [review]
👍

**Pull request merged**